### PR TITLE
fix(i18n): pass `build.format` when computing the redirect

### PR DESCRIPTION
**Pull request by ematipico**
## Changes

This PR adds `build.format` to the SSR Manifest.

This information is eventually needed by the i18n middleware to correctly compute the redirect.

## Testing

Existing tests should pass

## Docs

N/A

### CODE REVIEW

**Comment by lilnasy** [comment]
Could you elaborate on how `build.format` affects deployed paths?

**Comment by matthewp** [comment]
Can a test be added for this? Does it fix an existing known bug?

File: .changeset/fifty-pots-greet.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+The i18n redirects should consider the `build.format` configuration
```
Comment (florian-lefebvre) [inline .changeset/fifty-pots-greet.md:5]:
```suggestion
Makes i18n redirects take the `build.format` configuration into account
```

Comment (natemoo-re) [reply .changeset/fifty-pots-greet.md:5]: Thanks for breaking the `tense`ion Florian! 😆

**Comment by ematipico** [comment]
> Could you elaborate on how `build.format` affects deployed paths?

https://docs.astro.build/en/reference/configuration-reference/#buildformat

**Review by bluwy — APPROVED** [review]

**Pull request merged**